src/KOKKOS/meam_dens_init_kokkos.h

```diff
@@ -43,11 +43,23 @@ void MEAMKokkos<DeviceType>::operator()(TagMEAMZero, const int &i) const {
   d_rho0[i] = 0.0;
   d_arho2b[i] = 0.0;
   d_arho1(i,0) = d_arho1(i,1) = d_arho1(i,2) = 0.0;
-  for (int j = 0; j < 6; j++)
+  if (this->msmeamflag) {
```

Comment:
This will crash on GPUs (need to get rid of `this->` since it is a host pointer)

Reply:
Thanks @stanmoore1 .
Does this also apply to the Kokkos pair cpp files?
E.g. found another one here: https://github.com/rohskopf/lammps/blob/2f8ba9dd324aecdba9cd2426203bf777642b78ec/src/KOKKOS/pair_meam_ms_kokkos.cpp#L23

Reply:
Oops never mind, I see why `this->` is necessary in the wrapper.

Reply:
Yes just can't use `this->` inside a Kokkos parallel region, e.g. `parallel_for`